src/status_im/contexts/centralized_metrics/events.cljs

```diff
@@ -45,3 +45,7 @@
                           #(rf/dispatch [:profile.login/login-with-biometric-if-available
                                          (get-in db [:profile/login :key-uid])]))
               :shell?   true}]]]})))
+
+;; Events do nothing but they will be intercepted and tracked 
+(rf/reg-event-fx :centralized-metrics/navigated-to-collectibles-tab (fn []))
```

Comment:
We have a choice - create events under specific namespace, like `wallet/navigated-to-collectibles-tab`, or under `centralized-metrics`. I selected the second one for the sake of code readability. Developer will immediately see that event exists for metrics.

Reply:
Hey, we should not be introducing new events. We can just hook in the `:navigate-to` event that already exists. 

The idea with interceptors is to make the metrics layer invisible. We don't emit metric specific events, we just read other events, and track some to compute metrics.